RedacteurPortaal.Tests/DtoBuilder.cs

```diff
@@ -10,28 +10,51 @@ namespace RedacteurPortaal.Tests;
 
 public static class DtoBuilder
 {
+    public static CreateArchiveRequest BuildAddArchiveRequest()
+    {
+        return new CreateArchiveRequest() {
+            Title = "Title",
+            Label = "Label",
+            MediaPhotoItems = new(),
```

Comment:
Als je alle lists of hun standaard waarde laat (empty, niet null) dan test je ze eigenlijk niet, je hebt geen idee hoe hun inhoud verwerkt zou worden

Reply:
is nu gevuld met een inhoud.